docker/dev/docker-compose.yml

```diff
@@ -3,17 +3,27 @@ services:
   db:
     image: postgres:9.6
     ports:
-      - "${DB_PORT:-5432}:5432"
+    - "${DB_PORT:-5432}:5432"
+  minio:
+    image: minio/minio:RELEASE.2018-10-06T00-15-16Z
+    volumes:
+    - minio_data:/export
+    ports:
+    - ${S3_PORT:-0}:9000
+    environment:
+      MINIO_ACCESS_KEY: accessKey1
+      MINIO_SECRET_KEY: verySecretKey1
+      MINIO_HTTP_TRACE: "/var/log/minio.log"
+    command: server minio_data
   zalenium:
     image: "dosel/zalenium"
-    container_name: zalenium
     hostname: zalenium
     tty: true
     volumes:
       - /tmp/videos:/home/seluser/videos
       - /var/run/docker.sock:/var/run/docker.sock
     ports:
-      - 4444:4444
+      - ${ZALENIUM_PORT:-4444}:4444
```

Comment:
Can this be SELENIUM_PORT ?   Its an emerging standard !!  We are not interested in what software is providing the selenium service

Reply:
Fixed in 33ca404